beancount_import/matching.py

```diff
@@ -220,8 +227,17 @@ def _entry_and_posting_ids_key(entry: Transaction, mp: MatchablePosting):
 # Keyed by the sequence of source_posting ids.
 DatabaseValues = Dict[SourcePostingIds, Tuple[Transaction, MatchablePosting]]
 
+SearchPosting = NamedTuple('SearchPosting', [
+    ('number', Decimal),
+    ('key', Tuple),
```

Comment:
Please specify Tuple element types

Reply:
It's the output of _entry_and_posting_ids_key... I think that makes it a SourcePostingIds.